client/src/common/types.tsx

```diff
@@ -1,3 +1,7 @@
+export type TruthySunRequirements = 'Full Sun' | 'Partial Sun' | 'Full Shade';
+
+export type sun_requirements = TruthySunRequirements | '' | null;
```

Comment:
PascalCase is the standard for types :)

```suggestion
export type SunRequirements = TruthySunRequirements | '' | null;
```

Reply:
changed